packages/sanity/src/components/previews/portableText/BlockImagePreview.tsx

```diff
@@ -43,25 +43,27 @@ export function BlockImagePreview(props: BlockImagePreviewProps) {
       <Stack>
         <HeaderFlex paddingLeft={2} paddingRight={1} paddingY={1}>
           <Stack flex={1} space={2}>
-            <Text size={1} textOverflow="ellipsis" weight="semibold">
-              {title || fallbackTitle}
-            </Text>
+            {(title || fallbackTitle) && (
+              <Text size={1} textOverflow="ellipsis" weight="semibold">
+                {title ? renderPreviewNode(title as any, 'block') : fallbackTitle}
+              </Text>
+            )}
 
             {subtitle && (
               <Text muted size={1} textOverflow="ellipsis">
-                {renderPreviewNode(subtitle, 'block')}
+                {renderPreviewNode(subtitle as any, 'block')}
               </Text>
             )}
           </Stack>
 
           <Flex gap={1} paddingLeft={1}>
             {status && (
               <Box paddingX={2} paddingY={3}>
-                {renderPreviewNode(status, 'block')}
+                {renderPreviewNode(status as any, 'block')}
               </Box>
             )}
 
-            {actions}
+            {actions as React.ReactNode}
```

Comment:
If we just cast here, then doesnt that mean that actions never can be componentType? Why not change the actions type def?

Reply:
As mentioned in another comment I'll change this to `any` as it needs a follow up fix with a proper typings fix later. For now we just need to be able to land the react v18 PR and stop further scope creep 😬